test/e2e/tests/multichain-accounts/multichain-wallet-details.spec.ts

```diff
@@ -1,83 +1,48 @@
-import { Suite } from 'mocha';
 import { Mockttp } from 'mockttp';
+import { Suite } from 'mocha';
+import { withFixtures } from '../../helpers';
 import AccountListPage from '../../page-objects/pages/account-list-page';
-import WalletDetailsPage from '../../page-objects/pages/wallet-details-page';
 import { Driver } from '../../webdriver/driver';
 import HeaderNavbar from '../../page-objects/pages/header-navbar';
-import { withSolanaAccountSnap } from '../solana/common-solana';
-import { mockMultichainAccountsFeatureFlagStateTwo } from './common';
-
-// eslint-disable-next-line
-describe.skip('Multichain Accounts - Wallet Details', function (this: Suite) {
-  it('should view wallet details with one Ethereum and one Solana account and show SRP backup reminder', async function () {
-    await withSolanaAccountSnap(
-      {
-        title: this.test?.fullTitle(),
-        state: 2,
-        numberOfAccounts: 1,
-        withFixtureBuilder: (builder) =>
-          builder.withKeyringControllerMultiSRP().withPreferencesController({
-            dismissSeedBackUpReminder: false,
-          }),
-        withCustomMocks: async (mockServer: Mockttp) => {
-          return mockMultichainAccountsFeatureFlagStateTwo(mockServer);
-        },
-      },
-      async (driver: Driver) => {
-        const headerNavbar = new HeaderNavbar(driver);
-        await headerNavbar.openAccountMenu();
-
-        const accountListPage = new AccountListPage(driver);
-        await accountListPage.checkPageIsLoaded();
-
-        await accountListPage.checkWalletDetailsButtonIsDisplayed();
-        await accountListPage.clickWalletDetailsButton();
-
-        const walletDetailsPage = new WalletDetailsPage(driver);
-        await walletDetailsPage.checkPageIsLoaded();
+import FixtureBuilder from '../../fixtures/fixture-builder';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import { mockPriceApi } from '../tokens/utils/mocks';
 
-        await walletDetailsPage.checkWalletNameIsDisplayed('Wallet 1');
-        await walletDetailsPage.checkBalanceIsDisplayed('$5,643.50');
-        await walletDetailsPage.checkAccountIsDisplayed('Account 1');
-        await walletDetailsPage.checkAccountIsDisplayed('Solana 1');
-      },
-    );
-  });
-
-  it('should add new Ethereum account from wallet details', async function () {
-    await withSolanaAccountSnap(
+describe('Multichain Accounts - Wallet Details', function (this: Suite) {
+  it('should view wallet details with one Ethereum', async function () {
```

Comment:
This test is now completely different and it does not test the wallet details at all 😅